app/user_api/service.py

```diff
@@ -0,0 +1,57 @@
+# This file contains service functions for the user API
+from app.user_api.User import User
+from app.user_api.errors import UserNotFoundError, UserAuthenticationError
+from firebase_admin import firestore
+
+users_ref = firestore.client().collection('users')
+
+def findUser(email):
+    '''Given email, find and return corresponding User. Raises UserNotFoundError if user not found'''
+    return User.get(users_ref, email)
+
+def updateUser(user):
+    '''Push updated User object to Firestore'''
+    user.push()
+
+def getPoints(user):
+    '''Returns user's reward points balance'''
+    return user.reward_point_balance
+
+def getPOIFrequency(user):
+    '''Return POI object '''
+    pass    # TODO
+
+def getNumLinesParticipated(user):
+    '''Returns number of lines participated in by a User'''
+    return user.num_lines_participated
+
+def getTotalLineTime(user):
+    '''Return total time in line spent by User as an integer'''
+    return user.time_in_line
+
+def signInUser(user):
```

Comment:
I dont think you need to worry about this and signOut that's all handled in the front end